.github/workflows/on-release.yml

```diff
@@ -0,0 +1,49 @@
+name: Push Updated Problem Bank Scripts to Dependent Repos
+
+on: 
+  release:
+    types: [published]
+
+jobs:
+  get-version:
+    runs-on: ubuntu-latest
+    env:
+      TAG_NAME: ${{ github.event.release.tag_name }}
+    outputs:
+        ver: ${{ steps.version_name.outputs.version }}
+    steps:
+      - name: Strip leading 'v' from tag and return as output
+        id: version_name
+        run: echo "version=$(echo ${TAG_NAME/#v/})" >> $GITHUB_OUTPUT
+  push-to-dependents:
+    needs: get-version
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false # if one repo fails, continue with the others, it might be unrelated
+      matrix:
+        repo: [PrairieLearnUBC/pl-ubc-opb000, PrairieLearnUBC/pl-ubc-opb100] # add more repos here
+        # if the token needs to be different per repo, add a token matrix using the extend matrix syntax
+        # as shown here: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-expanding-configurations
+    steps:
+      - name: Checkout ${{ matrix.repo }}
+        uses: actions/checkout@v3 
+        with: # checks out default branch
+          repository: ${{ matrix.repo }}
+          token: ${{ secrets.PAT_GITHUB_PRAIRIELEARN }}
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+          # saves pip cache between runs, in case a run is ever finished before some runs are started and it can be reused somehow
+          cache: 'pip'
+      - name: Update PBS version in serverFilesCourse
+        run: |
+          rm -rf serverFilesCourse/problem_bank_* # cleanup/remove old versions
```

Comment:
```suggestion
          rm -rf serverFilesCourse/problem_bank_helpers # cleanup/remove old versions
          rm -rf serverFilesCourse/sigfig
          rm -rf serverFilesCourse/sortedcontainers
```

Let's be more explicit about the ones we're removing.

Reply:
If you don't mind, I'd like to leave the asterisk at the end, or add extra lines to catch the extra  `dist-info` folders so they don't pile up, otherwise they will over time since pip won't remove old ones when installing to a target directory

Reply:
Hmm good point.

My goal was just to future proof this in case someone happens to have a problem_bank_x package installed that will be removed by this command and not replaced.

Reply:
Oh if you're suggesting, 
```suggestion
          rm -rf serverFilesCourse/problem_bank_helpers*
          rm -rf serverFilesCourse/sigfig*
          rm -rf serverFilesCourse/sortedcontainers*
```
?

Then yes, that sounds good to me.
<img width="323" alt="Screenshot 2023-07-24 at 6 14 39 PM" src="https://github.com/open-resources/problem_bank_helpers/assets/2507459/0bf8e76f-5629-4033-a37a-3c9997fa3f2e">

Reply:
Something along those lines yes, maybe we can do the following which should be safe but more verbose:
```suggestion
          rm -rf serverFilesCourse/problem_bank_helpers
          rm -rf serverFilesCourse/problem_bank_helpers-*.dist-info
          rm -rf serverFilesCourse/sigfig
          rm -rf serverFilesCourse/sigfig-*.dist-info
          rm -rf serverFilesCourse/sortedcontainers
          rm -rf serverFilesCourse/sortedcontainers-*.dist-info
```

Testing it locally using `ls -l` I get the following for problem_bank_helpers:

```bash
ls -l problem_bank_helpers-*.dist-info
problem_bank_helpers-0.1.13.dist-info:
total 24
-rw-r--r-- 1 gavin gavin    4 Jun 29 16:33 INSTALLER
-rwxr-xr-x 1 gavin gavin    1 Jun 29 16:33 LICENSE
-rw-r--r-- 1 gavin gavin 2078 Jun 29 16:33 METADATA
-rw-r--r-- 1 gavin gavin 1555 Jun 29 16:33 RECORD
-rw-r--r-- 1 gavin gavin    0 Jun 29 16:33 REQUESTED
-rw-r--r-- 1 gavin gavin   88 Jun 29 16:33 WHEEL
-rw-r--r-- 1 gavin gavin  124 Jun 29 16:33 direct_url.json

problem_bank_helpers-0.1.14.dist-info:
total 20
-rw-r--r-- 1 gavin gavin    4 Jul  6 19:07 INSTALLER
-rwxr-xr-x 1 gavin gavin    1 Jul  6 19:07 LICENSE
-rw-r--r-- 1 gavin gavin 2027 Jul  6 19:07 METADATA
-rw-r--r-- 1 gavin gavin 1343 Jul  6 19:07 RECORD
-rw-r--r-- 1 gavin gavin   83 Jul  6 19:07 WHEEL
```
That would be a very strange glob for other packages to match I think

Reply:
Sounds good - let's do that:

```suggestion
          rm -rf serverFilesCourse/problem_bank_helpers serverFilesCourse/problem_bank_helpers-*.dist-info
          rm -rf serverFilesCourse/sigfig serverFilesCourse/sigfig-*.dist-info
          rm -rf serverFilesCourse/sortedcontainers sortedcontainers-*.dist-info
```